dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -422,6 +445,14 @@ def RunTool(cdm: Optional[CursesDisplayManager] = None) -> bool:
 
   tool.state.LogExecutionPlan()
 
+  try:
+    tool.ValidateArguments()
+  except errors.RecipeArgsValidatorError as exception:
+    if cdm:
+      cdm.EnqueueMessage('dftimewolf', str(exception), True)
+    logger.critical(str(exception))
+    return False
```

Comment:
I'd move this up right before LogExecutionPlan()

Reply:
Done.